contracts/examples/multisig/src/multisig.rs

```diff
@@ -25,9 +25,8 @@ pub trait Multisig:
         let board_vec = board.to_vec();
         let new_num_board_members = self.add_multiple_board_members(board_vec);
 
-        let num_proposers = self.num_proposers().get();
         require!(
-            new_num_board_members + num_proposers > 0,
+            new_num_board_members > 0,
```

Comment:
side-by fix. `num_proposers` is always 0 on `init`

Reply:
That condition is purposefully added to make the constructor upgrade-friendly. The `init` function is also called when upgrading, if anyone tries to upgrade it's likely they won't think about changing the constructor.